app/models/travel_advice_edition.rb

```diff
@@ -79,13 +80,17 @@ class << self; attr_accessor :fields_to_clone end
 
     state :published do
       validate :cannot_edit_published
-      validates :change_description, presence: { unless: :minor_update, message: "can't be blank on publish" }
+      validates :change_description, presence: { unless: :is_minor_update?, message: "can't be blank on publish" }
     end
     state :archived do
       validate :cannot_edit_archived
     end
   end
 
+  def is_minor_update?
+    minor_update || update_type == "minor"
```

Comment:
```suggestion
    minor_update || minor?
```

Minor: you can use [an enum](https://github.com/alphagov/content-publisher/blob/master/app/models/status.rb#L16) here; this could wait until we've got rid of the original column, so it's not too confusing.

Reply:
Am I able to use Rails' enum even though Mongoid doesn't have an `enum` type?

Reply:
Oops, you're right - I am thinking of Postgres integer/enum.